concordium-sdk/src/main/java/com/concordium/sdk/exceptions/InstanceNotFoundException.java

```diff
@@ -0,0 +1,34 @@
+package com.concordium.sdk.exceptions;
+
+import com.concordium.sdk.requests.getinstanceinfo.ContractAddress;
+import com.concordium.sdk.requests.getinstanceinfo.GetInstanceInfoRequest;
+import com.concordium.sdk.transactions.Hash;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+public class InstanceNotFoundException  extends Exception {
+
+    /**
+     * Block hash at which the instance information is to be fetched.
```

Comment:
I think you mean something like "... the smart contract could **not** be found."

Reply:
done